protocol/v2/ssv/validator/non_committee_validator.go

```diff
@@ -0,0 +1,69 @@
+package validator
+
+import (
+	"fmt"
+	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv/ibft/storage"
+	qbftcontroller "github.com/bloxapp/ssv/protocol/v2/qbft/controller"
+	"github.com/bloxapp/ssv/protocol/v2/types"
+	"go.uber.org/zap"
+)
+
+type NonCommitteeValidator struct {
+	logger         *zap.Logger
+	Share          *types.SSVShare
+	Storage        *storage.QBFTStores
+	qbftController *qbftcontroller.Controller
+}
+
+func NewNonCommitteeValidator(identifier spectypes.MessageID, opts Options) *NonCommitteeValidator {
+	// currently, only need domain & storage
+	config := &types.Config{
+		Domain:  types.GetDefaultDomain(),
+		Storage: opts.Storage.Get(identifier.GetRoleType()),
+	}
+	ctrl := qbftcontroller.NewController(identifier[:], &opts.SSVShare.Share, types.GetDefaultDomain(), config)
+	if err := ctrl.LoadHighestInstance(identifier[:]); err != nil {
+		fmt.Printf("failed to load highest instance: %e", err)
```

Comment:
The `%e` verb is used for the scientific notation of a floating-point number. Did you mean `%v`? Also, `fmt` adds no newline at the end, can we use logger instead?